### PR TITLE
[Recognizer] Setup CD

### PR DESCRIPTION
**Pull request by mickel8**
After this PR, a new docker Recognizer docker image will be created, published and deployed every time we create a tag `recognizer-vX` where `X` can be anything e.g. `v0.1.0` `v0.1.0-32dfeg` `v0.1.0-dev`. 

Docker image can be pulled with `docker pull ghcr.io/elixir-webrtc/apps/recognizer:0.1.0` (so with `v` stripped) or using `latest`.

Deployed version is available under https://recognizer.elixir-webrtc.org

### CODE REVIEW

**Comment by LVala** [comment]
Btw, there's something wrong with the tab title on the deployed version
image

Wasn't it `Home <dot> Recognizer` before?

**Review by LVala — APPROVED** [review]
Is the `__CI__...` a convention?

File: .github/workflows/__CD__deploy-image.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+jobs:
+  deploy-image:
+    name: Deploy image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: ${{ inputs.app-name }}
+
```
Comment (sgfn) [inline .github/workflows/__CD__deploy-image.yml:30]: We don't really need that, do we? The tag is present in the `github` context, and the container image is pulled from the registry

**Comment by mickel8** [comment]
@LVala Github workflows cannot be nested so I prefixed them with `__CI__` and `__CD__` for some kind of sorting and easier navigation

**Comment by mickel8** [comment]
@LVala 

> Wasn't it Home <dot> Recognizer before?

Fixed

**Pull request merged**